packages/design-picker/src/components/design-picker-category-filter/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { NavigableMenu, MenuItem, VisuallyHidden } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import { useI18n } from '@wordpress/react-i18n';
+import type { Category } from '../../types';
+import type { ReactElement } from 'react';
```

Comment:
Seems like eslint doesn't complain the order of import type 🤔

Reply:
I've just tried rearranging them and the auto ordering is working, this is the order the auto-fix wants to put them in. It is surprising though ...